child_compassion/models/project_reservation.py

```diff
@@ -0,0 +1,42 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class ProjectReservation(models.Model):
+
+    _name = 'compassion.reservation'
+    _description = 'Project Reservation'
+
+    beneficiary_global_id = None
+    channel_name = fields.Char(string='Channel Name', required=True, size=64)
```

Comment:
Remove required property, size may not be useful as well. You are not obliged to put `string='Channel Name'`, as it will be this string by default. It is only helpful when you want to put another name than the field_name.